omod/src/main/compass/sass/allergy.scss

```diff
@@ -48,18 +54,18 @@
     display: table;
   }
 
-  .horizontal > * {
+  .horizontal>* {
     display: table-cell;
   }
 
-  form > div {
+  form>div {
```

Comment:
Same as above?

Reply:
@dkayiwa I think it was about my IDE which auto formats ....let me see in my settings